iree/test/BUILD

```diff
@@ -4,8 +4,20 @@
 # See https://llvm.org/LICENSE.txt for license information.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
+load("//iree:build_defs.oss.bzl", "iree_cmake_extra_content")
+
 package(
     default_visibility = ["//visibility:public"],
     features = ["layering_check"],
     licenses = ["notice"],  # Apache 2.0
 )
+
+# Test tools has implicit thread dependency from absl.
+iree_cmake_extra_content(
+    content = """
+if (NOT ${IREE_ENABLE_THREADING})
```

Comment:
link the absl TODO (makes it easier to grep the codebase for places to clean) + style nit

```suggestion
# TODO(#3848): Test tools has implicit thread dependency from absl.
iree_cmake_extra_content(
    content = """
if(NOT ${IREE_ENABLE_THREADING})
```

Reply:
Done.